ftl/core/preferences.ftl

```diff
@@ -46,3 +46,6 @@ preferences-daily-backups = Daily backups to keep:
 preferences-weekly-backups = Weekly backups to keep:
 preferences-monthly-backups = Monthly backups to keep:
 preferences-minutes-between-backups = Minutes between automatic backups:
+preferences-appearance = <b>Appearance</b>
+preferences-animations = Animations
```

Comment:
Since we're not currently stopping all of them (eg graphs screen), maybe "Reduce animations" or Apple's "Reduce motion" would be more accurate?